diff --git a/node_modules/@uniswap/v3-periphery-0.8/contracts/libraries/OracleLibrary.sol b/node_modules/@uniswap/v3-periphery-0.8/contracts/libraries/OracleLibrary.sol
index ca7872f..423e3a4 100644
--- a/node_modules/@uniswap/v3-periphery-0.8/contracts/libraries/OracleLibrary.sol
+++ b/node_modules/@uniswap/v3-periphery-0.8/contracts/libraries/OracleLibrary.sol
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity >=0.5.0 <0.9.0;
 
-import '@uniswap/v3-core/contracts/libraries/FullMath.sol';
-import '@uniswap/v3-core/contracts/libraries/TickMath.sol';
-import '@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol';
+import '@uniswap/v3-core-0.8/contracts/libraries/FullMath.sol';
+import '@uniswap/v3-core-0.8/contracts/libraries/TickMath.sol';
+import '@uniswap/v3-core-0.8/contracts/interfaces/IUniswapV3Pool.sol';
 
 /// @title Oracle library
 /// @notice Provides functions to integrate with V3 pool oracle
